ui/round/src/clock/clockView.ts

```diff
@@ -31,6 +31,9 @@ export function renderClock(ctrl: RoundController, player: game.Player, position
         running: isRunning,
         emerg: millis < clock.emergMs,
```

Comment:
Not sure why this line isn't triggering a redraw already. I verified that when a flip happens, we come here and determine `emerg` to be `false`, but for some reason snabbdom doesn't redraw it and leaves the element as-is with the `emerg` class applied. Not how I expected snabbdom to behave 🤔